src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -1,31 +1,34 @@
 import { NextFunction, Request, Response } from "express";
 import { Errors } from "config/errors";
+import { GitHubUserClient } from "~/src/github/client/github-user-client";
 
 // TODO: need to update this later with actual data later on
 const servers = [{ id: 1, server: "http://github.internal.atlassian.com", appName: "ghe-app" }, { id: 2, server: "http://github.external.atlassian.com", appName: "ghe-app-2" }];
-const repos = [{ id: 1, name: "first-repo", org: "org-1" }, { id: 2, name: "second-repo", org: "org-1"  }, { id: 3, name: "third-repo", org: "org-2" }];
 const branches = [{ id: 1, name: "first-branch" }, { id: 2, name: "second-branch" }, { id: 3, name: "third-branch" }];
 
 
 export const GithubCreateBranchGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const {
 		jiraHost,
-		githubToken
+		githubToken,
+		hostname
 	} = res.locals;
 
 	if (!githubToken) {
 		return next(new Error(Errors.MISSING_GITHUB_TOKEN));
 	}
 
+	const gitHubAppClient = new GitHubUserClient(githubToken, req.log, hostname);
```

Comment:
Theres a definite possibility that we change this user client to the installation client. But for now it does what we need so we can revisit that later on. More of a by the way than anything to action.

Reply:
We need a specific `githubInstallationId` for using `GitHubInstallationClient`. And I don't think we will have just one `githubInstallationId`. Can discuss and revisit later.